crux-test/test/crux/api_test.clj

```diff
@@ -81,9 +81,6 @@
   (t/testing "empty db"
     (t/is (.db *api*)))
 
-  (t/testing "syncing empty db"
-    (t/is (nil? (.sync *api* (Duration/ofSeconds 10)))))
```

Comment:
Maybe you could note why this isn't useful - we did have an issue with performing operations on an empty DB in the past, so it's reassuring to see this test

Reply:
think it was part of removing the sync-to-no-lag commit (I cherry picked a few), will re-instate.